app/src/main/res/layout/activity_chat.xml

```diff
@@ -2,6 +2,8 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:theme="@style/Hedvig.Theme.Chat"
```

Comment:
```suggestion
```

This should be specified in the `AndroidManifest.xml`.

If not automatically applied in the editor, check if the `tools:context` attribute is set 🙂